integrations/docker/images/stage-2/chip-build-efr32/Dockerfile

```diff
@@ -30,8 +30,8 @@ RUN git clone --depth=1 --single-branch --branch=2.8.2 https://github.com/Silico
     rm -rf .git \
     && : # last line
 
-# Clone WiSeConnect SDK 3.1.1 (b2c1cd0)
```

Comment:
This file shouldn't be changed.

Reply:
ohhh! This got changed with rebase, I will revert it.